Automattic/jetpack
# Add WordPress.com toolbar navigation module

_**Note: Do not merge before this is rebased and retargeted against `feature/settings-overhaul` branch**_

Ref: https://github.com/Automattic/jetpack/issues/6254
~Test with: D4478-code~ (deployed)

Improves the experience for Jetpack authors when they've transferred their WordPress site from WordPress.com to a self-hosted plan by providing a similar experience to the front-end navigation on WordPress.com.

#### Testing instructions:

1. Navigate to `General` tab of Jetpack Settings.
2. Enable WordPress.com Toolbar module.

![toolbar](https://cloud.githubusercontent.com/assets/1182160/23401481/275ef690-fda8-11e6-944a-41bc7a811326.png)

3. Verify that the core admin bar is replaced with the new WordPress.com Toolbar.
4. The menus should look and work the same as they do on WordPress.com.

![masterbar](https://cloud.githubusercontent.com/assets/1182160/22947175/4908d6de-f2fa-11e6-81d1-1739535a1196.png)



If you run into any issues while testing, please report them here for easier tracking: https://github.com/Automattic/jetpack/milestone/75
